airbyte-cdk/python/airbyte_cdk/singer/source.py

```diff
@@ -116,7 +116,7 @@ def discover(self, logger: AirbyteLogger, config_container) -> AirbyteCatalog:
 
     def read(
         self, logger: AirbyteLogger, config_container: ConfigContainer, catalog_path: str, state_path: str = None
-    ) -> Generator[AirbyteMessage, None, None]:
+    ) -> Iterator[AirbyteMessage, None, None]:
```

Comment:
To match the `Source` interface.

Reply:
can we make it simpler by changing both to `Iterable[AirbyteMessage]`?

Reply:
yeap. this is from the existing interface type.